### DISCUSSION TITLE
Aspire RabbitMQ Deploy to container app configuration "rabbitmq Cookie file /var/lib/rabbitmq/.erlang.cookie must be accessible by owner only container"

### DISCUSSION THREAD

**Discussion by davide-podda** [microsoft/aspire#6578] [Q&A]
I try to deploy my aspire project to Container apps envrionment using azd.
```
var messaging = builder.AddRabbitMQ("messaging")
                       .WithManagementPlugin()
                       .WithDataVolume(isReadOnly: false);
```

when my container with rabbitmq starts, I get the following error:
"rabbitmq Cookie file /var/lib/rabbitmq/.erlang.cookie must be accessible by owner only container" .

How can I handle this error in my program.cs?

**Comment by davide-podda** [comment]
@davidfowl do you have any advice? I try to set RABBITMQ_ERLANG_COOKIE but i don't have any idea to configure in correct way using apphost/program.cs

**Reply by davidfowl** [reply]
I have no idea what that error means.

https://stackoverflow.com/questions/56665419/how-to-fix-cookie-file-var-lib-rabbitmq-erlang-cookie-must-be-accessible-by-o

**Reply by davpodda** [reply]
Yes i already read it, and more article but i dont find any solution..
